src/Components/Server/src/ComponentHub.cs

```diff
@@ -136,7 +128,7 @@ public async ValueTask<string> StartCircuit(string baseUri, string uri)
                 // If the circuit fails to initialize synchronously we can notify the client immediately
                 // and shut down the connection.
                 Log.CircuitInitializationFailed(_logger, ex);
-                await NotifyClientError(Clients.Caller, "The circuit failed to initialize.");
+                await NotifyClientError(Clients.Caller, GetClientErrorMessage(ex, "The circuit failed to initialize."));
```

Comment:
We weren't respecting error details on this exception.

Reply:
We weren't doing that because there's no user code that runs inside this try/catch block 😆  

After the comment at L104, it was expected that everything would succeed unless something is seriously wrong.

It was intentional before your changes, and based on my other feedback, I'm not sure it needs to change.

Reply:
Ok, I'll revert it then and I'll add a comment here for reference.

Reply:
done